SmilesMiles/View/TaskListView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  TaskListView.swift
+//  SmilesMiles
+//
+//  Created by Pavan Sai Nallagoni on 2/15/23.
+//
+
+import SwiftUI
+
+struct TaskListView: View {
+    @State private var tasks: [Task] = []
+    @State private var newTask = ""
+
+    var body: some View {
+        VStack {
+            HStack {
+                TextField("New Task", text: $newTask)
+                Button(action: addTask) {
+                    Text("Add")
+                }
+            }
+            .padding()
+
+            List {
+                ForEach(tasks.sorted(by: { !$0.isCompleted && $1.isCompleted })) { task in
+                    HStack {
+                        Button(action: {
+                            if let index = tasks.firstIndex(of: task) {
+                                tasks[index].isCompleted.toggle()
+                                tasks.sort(by: { !$0.isCompleted && $1.isCompleted })
+                            }
+                        }
+                        )
+                        {
+                            Image(systemName: task.isCompleted ? "checkmark.circle.fill" : "circle")
+                                .foregroundColor(.green)
+                        }
```

Comment:
Elsewhere you have formatting like this:
```
                        Button(action: {
                             isPresentingNewTrip = true
                         }) {
                             Image(systemName: "plus.circle.fill")
                             ```
I would format the same way here.  That way it's consistent and also easier to read.